tests/Resources/img/.htaccess

```diff
@@ -0,0 +1,20 @@
+<IfModule mod_php5.c>
```

Comment:
I'm not sure this is needed 🤔 `tests` folder is removed from the ZIP built for production. So if this file aims to protected shops, it's not needed?

Reply:
mmm yes then I suppose.